erts/emulator/test/signal_SUITE.erl

```diff
@@ -255,6 +257,55 @@ busy_dist_exit_signal(Config) when is_list(Config) ->
     stop_node(OtherNode),
     ok.
 
+busy_dist_demonitor_signal(Config) when is_list(Config) ->
+    BusyTime = 1000,
+    {ok, BusyChannelNode} = start_node(Config),
+    {ok, OtherNode} = start_node(Config, "-proto_dist gen_tcp"),
```

Comment:
Now with `peer` landed, may I suggest to use `{ok, Peer, Node} = ?CT_PEER(["-proto_dist", "gen_tcp"]),` instead?

Reply:
Yes, I've already rewritten it like that in a branch made for merging into master. All of these branches that I've pushed to github are for merging into branches preceding master which lacks peer support, so they need to use the old functionality.